src/load/azext_load/data_plane/load_test/custom.py

```diff
@@ -0,0 +1,287 @@
+import os
+
+import requests
+from azext_load.data_plane.utils.utils import (
+    create_or_update_body,
+    download_file,
+    get_admin_data_plane_client,
+    upload_test_plan,
+    upload_configuration_files,
+)
+from azure.cli.core.azclierror import ValidationError
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def create_test(
+    cmd,
+    load_test_resource,
+    test_id,
+    display_name=None,
+    resource_group_name=None,
+    load_test_config_file=None,
+    test_description=None,
+    test_plan=None,
+    configuration_files=None,
+    engine_instances=None,
+    env=None,
+    secrets=None,
+    certificate=None,
+    key_vault_reference_identity=None,
+    subnet_id=None,
+    no_wait=False,
+):
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    body = {}
+    body = create_or_update_body(
+        test_id,
+        body,
+        load_test_config_file=load_test_config_file,
+        display_name=display_name,
+        test_description=test_description,
+        engine_instances=engine_instances,
+        env=env,
+        secrets=secrets,
+        certificate=certificate,
+        key_vault_reference_identity=key_vault_reference_identity,
+        subnet_id=subnet_id,
+    )
+    list_of_tests = client.list_tests()
+    for test in list_of_tests:
+        if test_id == test.get("testId"):
```

Comment:
I dont follow this. Since our client supports upsert (`create_or_update_test`), should we not throw this error?

Reply:
I am assuming, and have seen the convention: `az XYZ create` => calls `PUT XYZ` and `az XYZ update` => calls `PATCH XYZ` 

Although I understand why this disambiguation is needed

Reply:
The spec indicated that create and update should be different calls, which required this check. Can discuss on this offline